lib/repository.py

```diff
@@ -101,7 +101,7 @@ def run(self):
             scripts = []
             print("Downloading scripts...")
             for line in version_file:
-                scripts.append(line.strip('\n').split(','))
+                scripts.append(line.decode("utf-8").strip('\n').split(','))
```

Comment:
We can't keep `latin-1` as default as Python 3 keeps `utf-8` as default encoding for source and raises an error if that is changed.